project/dexter-core/src/java/com/samsung/sec/dexter/core/config/PlatzKeywordFile.java

```diff
@@ -0,0 +1,59 @@
+package com.samsung.sec.dexter.core.config;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Map;
+import com.google.common.base.Charsets;
+import com.google.common.io.Files;
+import com.google.gson.Gson;
+import com.samsung.sec.dexter.core.exception.DexterRuntimeException;
+
+public class PlatzKeywordFile {
+	private String keyword;
+	
+	public void loadFromFile(final File file){
+		Map<String, Object> keywordMap = getKeywordMap(file);
+		setFields(keywordMap);
+		
+	}
+	
+	protected Map<String, Object> getKeywordMap(final File keywordFile) {
+		if (keywordFile.exists() == false) {
+			throw new DexterRuntimeException("There is no " + DexterConfig.PLATZ_KEYWORD_FILENAME 
+					+ " file : " + keywordFile.getName());
+		}
+		final StringBuilder keywordJson = new StringBuilder(50);
+		try {
+			for (String content : Files.readLines(keywordFile, Charsets.UTF_8)) {
+				keywordJson.append(content);
+				if (content.indexOf("}") >= 0) {
+					break;
+				}
+			}
+			final Gson gson = new Gson();
+            @SuppressWarnings("unchecked")
+            Map<String, Object> keywordMap = gson.fromJson(keywordJson.toString(), Map.class);
+            
+            
+            return keywordMap;
```

Comment:
!ENTRY org.eclipse.core.jobs 4 2 2016-03-31 12:57:49.776
!MESSAGE An internal error occurred during: "Monitoring target keyword in sourceinsight".
!STACK 0
java.lang.NullPointerException
	at com.samsung.sec.dexter.core.config.PlatzKeywordFile.setFields(PlatzKeywordFile.java:48)
	at com.samsung.sec.dexter.core.config.PlatzKeywordFile.loadFromFile(PlatzKeywordFile.java:16)
	at com.samsung.sec.dexter.daemon.job.MonitorForPlatzKeywordFile.checkPlatzKeywordFileAndAnalyze(MonitorForPlatzKeywordFile.java:68)
	at com.samsung.sec.dexter.daemon.job.MonitorForPlatzKeywordFile.run(MonitorForPlatzKeywordFile.java:50)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)